output/single-feedstock-web-catalog.json

```diff
@@ -0,0 +1,48 @@
+[
+  {
+    "title": "CHIRPS-2.0 global daily",
+    "description": "Climate Hazards Group InfraRed Precipitation with Station data (CHIRPS)\n",
+    "maintainers": [
+      {
+        "name": "Julius Busecke",
+        "github": "jbusecke"
+      }
+    ],
+    "provenance": {
+      "providers": [
+        {
+          "name": "Julius",
+          "description": "Just a guy testing some recipes. Nothing to see here.",
+          "roles": [
+            "producer",
+            "licensor"
+          ],
+          "url": null
+        }
+      ],
+      "license": "Just a Test",
+      "license_link": null
+    },
+    "thumbnail": null,
+    "tags": [
+      "precipitation",
+      "zarr"
+    ],
+    "links": null,
+    "stores": [
+      {
+        "id": "chirps-global-daily",
+        "name": "CHIRPS-2.0",
+        "url": "https://nyu1.osn.mghpcc.org/leap-pangeo-pipeline/chirps_feedstock/chirps-global-daily.zarr",
+        "ncviewjs:rechunking": null,
+        "public": true,
+        "geospatial": true,
+        "xarray_open_kwargs": {
+          "engine": "zarr"
+        },
+        "last_updated": "2024-11-19T21:19:44.192735+00:00"
+      }
```

Comment:
@norlandrhagen, it appears the JSON output doesn't have the pyramid output. i'm looking into this...

Reply:
ty!

Reply:
this is a bug in `leap-stc/data-management-utils`... pushing a fix shortly

Reply:
- https://github.com/leap-stc/leap-data-management-utils/pull/66

Reply:
fixed!